go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grpc-ecosystem/grpc-gateway/v2
 
-go 1.24
```

Comment:
I think this change should be the only one necessary to allow the last two major versions, right? It's fine for us to use 1.24 in most of our own testing and so on. Could you revert the other changes please?

Reply:
Whether tests should be run on the minimum supported version of Go?

Reply:
We run builds for `stable` and `oldstable`, I think tests can keep running for Go 1.24. Please revert the other changes and I can merge this.